SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -40,6 +40,19 @@ bool USpatialClassInfoManager::TryInit(USpatialNetDriver* InNetDriver, UActorGro
 	return true;
 }
 
+bool USpatialClassInfoManager::CheckClassIsSupported(const FString& PathName)
+{
+	if (!IsSupportedClass(PathName))
```

Comment:
Realise this is just moved code, but we probably shouldn't be doing it in Shipping